packages/pytest-simcore/src/pytest_simcore/docker_compose.py

```diff
@@ -179,6 +170,15 @@ def ops_services_config_file(request, temp_folder, ops_docker_compose):
 
 
 # HELPERS ---------------------------------------------
+def _minio_fix(service_environs: Dict) -> Dict:
+    """this hack ensures that S3 is accessed from the host at all time, thus pre-signed links work.
+        172.17.0.1 is the docker0 interface, which redirect from inside a container onto the host network interface.
+    """
+    if "S3_ENDPOINT" in service_environs:
+        service_environs["S3_ENDPOINT"] = "172.17.0.1:9001"

```

Comment:
Do you set that interfaces IP address somewhere? I doubt that this is always the same address

Reply:
So to this question, that is true I am not sure if it will remain at all time.
But it seems docker for Mac and Windows have a special DNS defined inside containers called ```host.docker.internal``` that allows to access the host network interface from inside the container. The workaround under linux is to use that 172.17.0.1 IP. I tested it and it seems to work nicely on local/CI computers. see info [here](https://stackoverflow.com/questions/48546124/what-is-linux-equivalent-of-host-docker-internal)
But it is true that it might break at some point.